<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTab</class>
 <widget class="QTabWidget" name="MainTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="notesTab">
   <attribute name="title">
    <string>Dive Notes</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="3" column="1">
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>501</width>
         <height>535</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="KMessageWidget" name="diveNotesMessage" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="DateTimeLabel">
          <property name="text">
           <string>Starttime</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="TemperaturesLabel">
          <property name="text">
           <string>air / water Temperatures</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QDateTimeEdit" name="dateTimeEdit">
          <property name="displayFormat">
           <string>M/d/yy h:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="airWaterTempLayout">
          <item>
           <widget class="QLineEdit" name="airtemp">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="watertemp">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLineEdit" name="location">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="LocationLabel">
          <property name="text">
           <string>Location</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="TagLabel">
          <property name="text">
           <string>Tags</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QLineEdit" name="coordinates">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLineEdit" name="divemaster">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="CoordinatedLabel">
          <property name="text">
           <string>Coordinates</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="BuddyLabel">
          <property name="text">
           <string>Buddy</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="buddy">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="DivemasterLabel">
          <property name="text">
           <string>Divemaster</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="suit">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <layout class="QHBoxLayout" name="ratingVisibilityLabels">
          <item>
           <widget class="QLabel" name="RatingLabel">
            <property name="text">
             <string>Rating</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="visibilityLabel">
            <property name="text">
             <string>Visibility</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="14" column="0" colspan="2">
         <widget class="QTextEdit" name="notes">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="SuitLabel">
          <property name="text">
           <string>Suit</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <layout class="QHBoxLayout" name="ratingVisibilityWidgets">
          <item>
           <widget class="StarWidget" name="rating" native="true"/>
          </item>
          <item>
           <widget class="StarWidget" name="visibility" native="true"/>
          </item>
         </layout>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="NotesLabel">
          <property name="text">
           <string>Notes</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0" colspan="2">
         <widget class="QDialogButtonBox" name="notesButtonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
        <item row="12" column="0" colspan="2">
         <widget class="TagWidget" name="tagWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="equipmentTab">
   <attribute name="title">
    <string>Equipment</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>515</width>
         <height>473</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="KMessageWidget" name="diveEquipmentMessage" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="TableView" name="cylinders" native="true"/>
             <widget class="TableView" name="weights" native="true"/>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QDialogButtonBox" name="equipmentButtonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="infoTab">
   <attribute name="title">
    <string>Dive Info</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>515</width>
         <height>473</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Date</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="dateText">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_12">
          <property name="title">
           <string>Interval</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="surfaceIntervalText">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Gases Used</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="oxygenHeliumText">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Gas Consumed</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="gasUsedText">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>SAC</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="sacText">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Max. Depth</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="maximumDepthText">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>Ave. Depth</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="averageDepthText">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>OTU</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="otuText">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox_10">
          <property name="title">
           <string>Air Pressure</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="airPressureText">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>Air Temperature</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="airTemperatureText">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>Water Temperature</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="waterTemperatureText">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_11">
          <property name="title">
           <string>Dive Time</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="diveTimeText">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Salinity</string>
          </property>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QLabel" name="salinityText">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="statisticsTab">
   <attribute name="title">
    <string>Stats</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout1">
    <item row="0" column="1">
     <widget class="QScrollArea" name="scrollArea_4">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>515</width>
         <height>473</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBoxb">
          <property name="title">
           <string>Depth</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2b">
           <item>
            <widget class="MinMaxAvgWidget" name="depthLimits" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_8b">
          <property name="title">
           <string>Temperature</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5b">
           <item>
            <widget class="MinMaxAvgWidget" name="tempLimits" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_4b">
          <property name="title">
           <string>SAC</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10b">
           <item>
            <widget class="MinMaxAvgWidget" name="sacLimits" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_14">
          <property name="title">
           <string>Duration</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_12b">
           <item>
            <widget class="MinMaxAvgWidget" name="timeLimits" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_11b">
          <property name="title">
           <string>Total Time</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11b">
           <item>
            <widget class="QLabel" name="totalTimeAllText">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QGroupBox" name="groupBox_7b">
          <property name="title">
           <string>Dives</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="divesAllText">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
       <zorder>groupBox_8b</zorder>
       <zorder>groupBox_4b</zorder>
       <zorder>groupBox_11b</zorder>
       <zorder>groupBox_14</zorder>
       <zorder>groupBoxb</zorder>
       <zorder>groupBox_7b</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QWidget</extends>
   <header>kmessagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StarWidget</class>
   <extends>QWidget</extends>
   <header>starwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MinMaxAvgWidget</class>
   <extends>QWidget</extends>
   <header>simplewidgets.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QWidget</extends>
   <header>tableview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TagWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>qt-ui/tagwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
